test/sanity/validate-modules/validate-modules

```diff
@@ -686,9 +686,13 @@ class ModuleValidator(Validator):
             else:
                 self.warnings.append((312, 'No RETURN provided'))
         else:
-            _, errors, traces = parse_yaml(doc_info['RETURN']['value'],
+            data, errors, traces = parse_yaml(doc_info['RETURN']['value'],
                                            doc_info['RETURN']['lineno'],
                                            self.name, 'RETURN')
+
+            if data:
+                for ret_key in data:
+                    self._validate_docs_schema(data[ret_key], return_schema(data[ret_key]), 'RETURN', 319)
```

Comment:
Not sure if this is right. As this stands we get:
`ERROR:319:RETURN.returned: required key not provided @ data['returned']. Got None`

Which doesn't list which returned value it's talking about

Compare to `DOCUMENTATION` check
`ERROR:305:DOCUMENTATION.options.vlagArg1.ruired: extra keys not allowed @ data['options']['vlagArg1']['ruired']. Got True` notice the `options.vlagArg1`

I think the `return_schema` needs to apply to the top level, rather than been iterated foreach item in RETURN